### PR TITLE
530 popup card tweaks

### PR DESCRIPTION
**Pull request by AliciadW**
Fixes https://github.com/laws-africa/indigo/issues/530

### CODE REVIEW

**Comment by AliciadW** [comment]
@longhotsummer I've done a bit of digging and testing and it looks like this is the best solution. I've added another element to catch the `mouseleave` event. It works locally for me. I would like to test it out and see if we need to add another function to close all popovers if the user isn't hovering over any popover links. 
Unfortunately popover.js needs some customisation to allow the popover to stay open while hovering on it, which is why this approach has been taken.

**Comment by AliciadW** [comment]
@longhotsummer please leave this open for another day or two. I'm going to move on to the next issue, want want to revisit this tomorrow just to have a look at it with fresh eyes. :-)

File: indigo_app/templates/indigo_api/work_popup.html [hunk 1]
```diff
@@ -1,41 +1,24 @@
...
   <h6>{{ work.title }}</h6>
-  <p class="text-muted mb-1 wb-all">{{ work.frbr_uri }}</p>
+  <p class="text-muted mb-3 wb-all">{{ work.frbr_uri }}</p>
 
   {% if work.assent_date %}
-    <p class="mb-1">Assented to on {{ work.assent_date|date:"Y-m-d" }}</p>
+    <p class="mb-2">Assented to on {{ work.assent_date|date:"Y-m-d" }}.</p>
   {% endif %}
 
-  <p class="mb-1">Published on {{ work.publication_date|date:"Y-m-d" }}
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_popup.html]: Urg, please can we keep these on separate lines! The long line is crazy to review and debug. To handle the whitespace issue before the '.' at the end, move the {% if %} onto the end of the line above:

```
<p class="mb-1">Published on {{ work.publication_date|date:"Y-m-d" }}{%
  if work.publication_name and work.publication_number and work.publication_document %}
    in ... <a>aoeu</a>{%
  endif %}.
</p>
```

**Comment by longhotsummer** [comment]
Thanks @AliciadW -- I'll leave this open. I haven't tested it locally yet.

File: indigo_app/templates/indigo_api/work_popup.html (hunk 1 above)
Comment (AliciadW) [reply indigo_app/templates/indigo_api/work_popup.html]: Thanks :-)
Updated

**Comment by AliciadW** [comment]
Screenshot 2019-05-15 at 07 38 58

**Comment by longhotsummer** [comment]
I'm happy to merge this and see if it fixes the issue.

**Comment by AliciadW** [comment]
@longhotsummer please approve the changes. I resolved the merge conflict.

**Comment by longhotsummer** [comment]
Approved!

**Pull request merged**